app/controllers/podcast_metrics_controller.rb

```diff
@@ -3,11 +3,98 @@ class PodcastMetricsController < ApplicationController
 
   def show
     authorize @podcast, :show?
+
+    @episodes =
+      @podcast.episodes
+        .published
+        .order(published_at: :desc)
+        .paginate(params[:page], params[:per])
+    @date_start = 30.days.ago
+    @date_end = Time.zone.now
+
+    if clickhouse_connected?
```

Comment:
these first 4 are the `downloads` queries, separated into 4: episode downloads within date range, episode downloads all time, all downloads within date range, all downloads all time. double check on query selection and grouping?